concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-120	120	tiralíneas	drawing pen			Compound	3.57843137254902	10		
+Desrochers-2010-330-121	121	tocadiscos	record player			Compound	5.03968884955819	10	18	5.67
+Desrochers-2010-330-122	122	tragalibros	gadgets			Compound	1.47058823529412	11		
+Desrochers-2010-330-123	123	tragaperras	slots			Compound	5.63	11	11	
+Desrochers-2010-330-124	124	trotamundos	globetrotter			Compound	5.1296511783235	11	1	
+Desrochers-2010-330-125	125	agua	water	WATER	948	Noun	6.86274509803922	4	1654	6.8
+Desrochers-2010-330-126	126	año	year	YEAR	1226	Noun	6.56862745098039	3	1926	6.68
+Desrochers-2010-330-127	127	barro	mud	MUD	640	Noun	5.70588235294118	5	132	6.07
+Desrochers-2010-330-128	128	bebida	drink	ALCOHOL (DRINK)	2050	Noun	6.59803921568627	6	68	6.5
+Desrochers-2010-330-129	129	bosque	forest	FOREST	420	Noun	5.76470588235294	6	273	5.04
+Desrochers-2010-330-130	130	bota	boot	BOOT	585	Noun	6.08823529411765	4	29	6.35
+Desrochers-2010-330-131	131	botella	bottle	BOTTLE	1321	Noun	6.55882352941176	7	177	6.34
+Desrochers-2010-330-132	132	brisa	breeze			Noun	5	5	102	5.72
+Desrochers-2010-330-133	133	cabeza	head	HEAD	1256	Noun	6.42156862745098	6	1976	6.18
+Desrochers-2010-330-134	134	cabra	goat	GOAT	1502	Noun	5.23529411764706	5	63	5.94
+Desrochers-2010-330-135	135	caída	fall			Noun	5.74509803921569	5	254	6.45
+Desrochers-2010-330-136	136	caldo	broth	SOUP	1547	Noun	5.42156862745098	5	62	5.61
+Desrochers-2010-330-137	137	calle	street	STREET	1362	Noun	6.6078431372549	5	1340	6.8
+Desrochers-2010-330-138	138	cama	bed	BED	1663	Noun	6.73529411764706	4	764	6.41
+Desrochers-2010-330-139	139	cámara	camera	CAMERA	2618	Noun	6.00980392156863	6	223	6.16
+Desrochers-2010-330-140	140	carnet	card	CARD	2619	Noun	6.27450980392157	6	33	6.63
+Desrochers-2010-330-141	141	carta	letter	LETTER	1612	Noun	6.2843137254902	5	472	6.56
+Desrochers-2010-330-142	142	chimenea	chimney	CHIMNEY	1305	Noun	5.29712954133031	8	79	5.27
+Desrochers-2010-330-143	143	choque	shock			Noun	5.76470588235294	6	84	5.9
+Desrochers-2010-330-144	144	cielo	sky	SKY	1732	Noun	6.38235294117647	5	620	6.26
+Desrochers-2010-330-145	145	cigarro	cigar			Noun	6.2156862745098	7	45	5.71
+Desrochers-2010-330-146	146	circuito	circuit			Noun	5.46078431372549	8	106	4.55
+Desrochers-2010-330-147	147	coche	car	CAR	1111	Noun	6.77450980392157	5	688	6.59
+Desrochers-2010-330-148	148	cohete	rocket			Noun	5.11764705882353	6	36	5.65
+Desrochers-2010-330-149	149	corazón	heart	HEART	1223	Noun	6.22549019607843	7	848	6.76
+Desrochers-2010-330-150	150	corcho	cork			Noun	5.31372549019608	6	22	
+Desrochers-2010-330-151	151	costa	coast	COAST	1525	Noun	5.6078431372549	5	276	5.46
+Desrochers-2010-330-152	152	cristal	glass	GLASS	604	Noun	6.31372549019608	7	264	6.61
+Desrochers-2010-330-153	153	cuchillo	knife	KNIFE	1352	Noun	6.51960784313726	8	86	6.31
+Desrochers-2010-330-154	154	cuento	story	STORY	2060	Noun	5.98039215686275	6	275	6.46
+Desrochers-2010-330-155	155	culo	article			Noun	6.35294117647059	4	140	6.05
```

Comment:
culo is "ass": https://dees.dict.cc/?s=culo

Reply:
It is less common than in Portuguese, but "culo" can also sometimes mean only "anus".